### ISSUE TITLE
"DhtOp was rejected during system validation": private entries are not actually private.

### ISSUE THREAD

**Issue by guillemcordoba** [holochain/holochain#1100]
While testing the latest [release of Kizuna](https://github.com/hc-institute-japan/kizuna/releases/download/0.1.5/kizuna.webhapp) with the latest [release of the Launcher](https://github.com/holochain/launcher/releases/tag/v0.3.5) (Holochain v114), we get this error in the logs:

```
[2021-11-05T14:53:53.916609792+01:00] INFO - [HOLOCHAIN] [2mNov 05 14:53:53.916[0m [33m WARN[0m holochain::core::workflow::sys_validation_workflow: agent=uhCAkPcGf7Hf8ZLGCPbdU95dz5JdoE0XBOlCuYux1Tohh9vNs1an5 msg="DhtOp was rejected during system validation." op=StoreElement(703e0a4b63e500a55583e82994e42d596a05bbd3e33bb46a8ee5f02d06e4bbe3c8b7b28649ef5c01629ab5d6193d5c8714b96bc2b7c901bf88aa07ecac773f0d, Create(Create { author: AgentPubKey(uhCAk9oA350ohoDb-DJX2IwIxtYSmJrnQmurtc4PIznhKIYFfqVLQ), timestamp: Timestamp(2021-11-05T13:53:50.610851Z), header_seq: 9, prev_header: HeaderHash(uhCkkhyYt-7-vm-Kr4fXxdmyz_QNp_O6jWxmrRxGuoEZ8CeJcrCa4), entry_type: App(AppEntryType { id: EntryDefIndex(0), zome_id: ZomeId(3), visibility: Private }), entry_hash: EntryHash(uhCEkCRnY0MsrkCdDP1BAJuZA5CQS8Of5OyM4-vZss8BSc1gJPE-m) }), Some(App(AppEntryBytes({"typingIndicator":true,"readReceipt":true}))))
```

The messages don't get actually gossiped due to this error. I can see that the entry is supposed to be private, but it really isn't.

**Comment by Connoropolous** [comment]
related but not? #1101

**Comment by guillemcordoba** [comment]
I'll reopen this as we are seeing the exact same problem with Holochain v0.0.115 and launcher v0.3.6.

**Comment by abe-njama** [comment]
@guillemcordoba could you try again on develop especially now with the new db entries and let us know what the outcome is. [db refactor](https://github.com/holochain/holochain/pull/1130) has been merged into develop.

**Comment by guillemcordoba** [comment]
I can confirm this is fixed as per latest holochain release v0.0.119, Closing this.
